commit c030961fde642e82d764c16e2a7b2b8a85b3eacc
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-04-16T14:21:06-04:00 (6 months ago)
    
    TOOLS-2242 sdcnode builds for minimal-64-19.1.0
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/nodeconfigs.json b/nodeconfigs.json
index 5582f7b..2f2fcf5 100644
--- a/nodeconfigs.json
+++ b/nodeconfigs.json
@@ -1,4 +1,33 @@
 [
+    {
+        "// image": "minimal-64 19.1.0",
+        "image": "fbda7200-57e7-11e9-bb3a-8b0b548fcc37",
+        "version": "v8.15.1",
+        "patch_files": [
+            "patches/http-keepAliveTimeout-default-0-v8.x.patch",
+            "patches/tls-reload-certs-v8.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "gcc": "/opt/local/gcc7/bin/gcc",
+        "cflags": "-fno-aggressive-loop-optimizations",
+        "rpath": "$ORIGIN/../lib",
+        "include_gcc_runtime_libs": true,
+        "build_tag": "gz"
+    },
+    {
+        "// image": "minimal-64 19.1.0",
+        "image": "fbda7200-57e7-11e9-bb3a-8b0b548fcc37",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v8.15.1",
+        "patch_files": [
+            "patches/http-keepAliveTimeout-default-0-v8.x.patch",
+            "patches/tls-reload-certs-v8.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "gcc": "/opt/local/gcc7/bin/gcc",
+        "cflags": "-fno-aggressive-loop-optimizations",
+        "build_tag": "zone64"
+    },
     {
         "// image": "minimal-64-lts 18.4.0",
         "image": "c2c31b00-1d60-11e9-9a77-ff9f06554b0f",
@@ -122,6 +151,37 @@
         "gcc": "/opt/local/gcc49/bin/gcc",
         "build_tag": "zone"
     },
+    {
+        "// image": "minimal-64 19.1.0",
+        "image": "fbda7200-57e7-11e9-bb3a-8b0b548fcc37",
+        "version": "v6.17.0",
+        "patch_files": [
+            "patches/http-keepAliveTimeout-default-0-v6.17+.patch",
+            "patches/tls-reload-certs-v6.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "gcc": "/opt/local/gcc7/bin/gcc",
+        "cflags": "-fno-aggressive-loop-optimizations",
+        "rpath": "$ORIGIN/../lib",
+        "include_gcc_runtime_libs": true,
+        "npm_version": "v6.4.1",
+        "build_tag": "gz"
+    },
+    {
+        "// image": "minimal-64 19.1.0",
+        "image": "fbda7200-57e7-11e9-bb3a-8b0b548fcc37",
+        "comment": "For use by services running in zones. Statically link to OpenSSL and zlib *for now* until OS-1294/TOOLS-130.",
+        "version": "v6.17.0",
+        "patch_files": [
+            "patches/http-keepAliveTimeout-default-0-v6.17+.patch",
+            "patches/tls-reload-certs-v6.x.patch"
+        ],
+        "configure_flags": "--without-snapshot --dest-cpu=x64 --with-dtrace",
+        "gcc": "/opt/local/gcc7/bin/gcc",
+        "cflags": "-fno-aggressive-loop-optimizations",
+        "npm_version": "v6.4.1",
+        "build_tag": "zone64"
+    },
     {
         "// image": "minimal-64-lts 18.4.0",
         "image": "c2c31b00-1d60-11e9-9a77-ff9f06554b0f",
